hilti/toolchain/include/compiler/detail/optimizer/pass-id.h

```diff
@@ -19,6 +19,7 @@ enum PassID {
     DeadCodeCFG,
     ConstantPropagation,
     RemoveUnusedParameters,
+    FlattenBlocks,
```

Comment:
Is this the best position in the order of passes? Asking because the commit message says  that other passes will benefit from having blocks flattened. Should we do it earlier, instead of waiting for the next iteration?